src/Esfa.Vacancy.Register.Api/Web.config

```diff
@@ -17,7 +17,7 @@
     <add key="EnvironmentName" value="DEV" />
     <add key="LoggingRedisKey" value="logstash" />
     <add key="LoggingRedisConnectionString" value="" />
-    <add key="AVMSPLUSCONNECTIONSTRING" value="" />    
+    <add key="DasApprenticeshipInfoApiBaseUrl" value="http://das-pp-apprenticeshipinfoservice.cloudapp.net/" />
```

Comment:
Also you should use the environment variables instead of putting that environment in a public repo.

```powershell
[Environment]::SetEnvironmentVariable("DAS_DasApprenticeshipInfoApiBaseUrl", "http://das-pp-apprenticeshipinfoservice.cloudapp.net/", "User")
```